chromium_src/content/public/browser/browser_context.h

```diff
@@ -6,11 +6,14 @@
 #ifndef BRAVE_CHROMIUM_SRC_CONTENT_PUBLIC_BROWSER_BROWSER_CONTEXT_H_
 #define BRAVE_CHROMIUM_SRC_CONTENT_PUBLIC_BROWSER_BROWSER_CONTEXT_H_
 
-#define IsOffTheRecord IsTor() const; \
-    virtual bool IsOffTheRecord
+#define IsOffTheRecord \
```

Comment:
what is this change? Doesn't seem related and also should have 4 space indent for continuation of a line

Reply:
There have been changes to the lint script which mean that the CI fails unless you do a full format. This will possibly be resolved when https://github.com/brave/brave-core/pull/7402 lands.

Reply:
In the meantime, I can remove these full format changes.